site/layouts/partials/home/components-utilities.html

```diff
@@ -0,0 +1,88 @@
+<section class="pb-md-5 mb-5">
+  <div class="col-lg-8 mb-5">
+    <div class="masthead-followup-icon d-inline-block mb-3 me-2" style="--bg-rgb: var(--bs-danger-rgb);">
+      <svg class="bi fs-1"><use xlink:href="#menu-button-wide-fill"></use></svg>
+    </div>
+    <svg class="bi me-2 fs-2 text-muted"><use xlink:href="#plus"></use></svg>
+    <div class="masthead-followup-icon d-inline-block mb-3" style="--bg-rgb: var(--bs-info-rgb);">
+      <svg class="bi fs-1"><use xlink:href="#braces-asterisk"></use></svg>
+    </div>
+    <h2 class="display-5 mb-3 fw-semibold lh-sm">Components, meet the Utility&nbsp;API</h2>
+    <p class="lead fw-normal">
+      New in Bootstrap 5, our utilities are now generated by our <a href="/docs/{{ .Site.Params.docs_version }}/utilities/api/">Utility API</a>. We built it as a feature-packed Sass map that can be quickly and easily customized. It's never been easier to add, remove, or modify any utility classes. Make utilities responsive, add pseudo-class variants, and give them custom names.
+    </p>
+  </div>
+  <div class="row gx-md-5">
+    <div class="col-lg-6">
+      <h3 class="fw-semibold">Quickly customize components</h3>
+      <p>Apply any of our included utility classes to our components to customize their appearance, like the navigation example below. There are hundreds of classes available—from <a href="/docs/{{ .Site.Params.docs_version }}/utilities/position/">positioning</a> and <a href="/docs/{{ .Site.Params.docs_version }}/utilities/sizing/">sizing</a> to <a href="/docs/{{ .Site.Params.docs_version }}/utilities/colors/">colors</a> and <a href="/docs/{{ .Site.Params.docs_version }}/utilities/shadows/">effects</a>.</p>
+      <div class="p-4 border rounded-3 mb-4">
```

Comment:
I liked the previous version where you could play with the components directly in this page by clicking on the different items. Have you removed those interactions in order to have a simpler code really showing the customization?
What can be a little bit strange is that you can click in the Dropdown in the other section and not in this one.